arenadata/readme.md

```diff
@@ -32,6 +32,27 @@ We need to execute [../concourse/scripts/ic_gpdb.bash](../concourse/scripts/ic_g
 
 To use gdb inside the container, add the `--privileged` flag to the run command.
 
+## Jit regression tests suite
+
+* jit tests are basically no different from regular regression tests except they are executed with jit enabled
+* jit tests need to be executed with optimizer both on and off. Notice that make flags differ a bit for each scenario
+
+* optimizer=on
+```bash
+ docker run --name gpdb7_opt_on --rm -it -e TEST_OS=centos \
```

Comment:
Target OS for GPDB 7 is rocky8. Maybe, it should be change here.

Reply:
Done